Develop

jarkkokoivunen opened the pull request:
Added Query -task that can be used for mysql queries. Options to extract resultset into xml, json or csv.

narnil commented:
Add .gitignore and remove unnecessary files and folders such as .vs/, bin/ and obj/

jarkkokoivunen commented:
> Add .gitignore and remove unnecessary files and folders such as .vs/, bin/ and obj/

Added .gitignore and removed unnecessary files and folders. Can you check is it ok now?

narnil commented on Frends.Community.MySql/Definitions.cs:
```diff
@@ -0,0 +1,203 @@
+    /// <summary>
+    /// Enumerator representing MySql parameter data types
+    /// </summary>
+    public enum QueryParameterType
```
Is this required? MySqlDbType enum seems to be exactly the same. Can it be used directly?

jarkkokoivunen replied:
You were right, it was not required. Now using MySqlDbType directly.

narnil commented on .gitignore:
```diff
@@ -0,0 +1,4 @@
+.vs
+bin
+obj
+/nuspec/*.nupkg
```
Add *.user

jarkkokoivunen replied:
*.user added

jarkkokoivunen commented on Frends.Community.MySql/Frends.Community.MySql.csproj:
```diff
@@ -0,0 +1,32 @@
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Company>HiQ Finland</Company>
+    <Description>Task for performing oracle queries</Description>
```
This is changed

jarkkokoivunen commented on Frends.Community.MySql/Extensions.cs:
```diff
@@ -0,0 +1,213 @@
+                            // add row element value
+                            switch (reader.GetDataTypeName(i))
+                            {
+                                case "Decimal":
```
Oracle task was used as a base for this MySql task, so that's why this was there. I removed it, but has to be tested if it works.
